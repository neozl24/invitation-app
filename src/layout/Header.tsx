import broccoliLogo from '/broccoli.svg'

export const Header = () => {
  return (
    <div className='w-full h-20 sm:h-24 flex justify-center border-b border-solid border-b-neutral-300'>
      <div className='flex h-full items-center justify-center w-full lg:w-[1000px] lg:justify-start'>
        <img src={broccoliLogo} width='32' alt="Broccoli logo" className='mr-4' />
        <h3 className='font-bold text-neutral-500 text-3xl sm:text-4xl'>BROCCOLI & CO</h3>
      </div>
    </div>
  )
}
