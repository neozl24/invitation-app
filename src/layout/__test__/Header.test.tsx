import { render } from '@testing-library/react'
import { Header } from '../Header'

test('renders name', () => {
  const { asFragment, getByText } = render(<Header />)

  expect(getByText('BROCCOLI & CO')).toBeDefined()
  expect(asFragment()).toMatchSnapshot()
})
