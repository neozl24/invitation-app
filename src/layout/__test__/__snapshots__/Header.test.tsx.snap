// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders name 1`] = `
<DocumentFragment>
  <div
    class="w-full h-16 sm:h-24 flex justify-center border-b border-solid border-b-neutral-300"
  >
    <div
      class="flex h-full items-center justify-center w-full lg:w-[1000px] lg:justify-start"
    >
      <img
        alt="Broccoli logo"
        class="mr-4 hidden sm:inline"
        src="/broccoli.svg"
        width="32"
      />
      <h3
        class="font-bold text-neutral-500 text-3xl sm:text-4xl"
      >
        BROCCOLI & CO
      </h3>
    </div>
  </div>
</DocumentFragment>
`;
